lib/rules/no-unsupported-features/es-syntax.json

```diff
@@ -0,0 +1,607 @@
+{
+    "no-accessor-properties": {
+        "supported": ">=0.10.0"
```

Comment:
I changed "supported" to always be a `semver.Range` it makes the comparisons much simpler. I am also thinking about changing the node-builtins to do the same :thinking:

But that is a later monster :joy: